### DISCUSSION TITLE
tar_download: no ETag or Last-Modified for url

### DISCUSSION THREAD

**Discussion by jimjam-slam** [ropensci/tarchetypes#114] [Help]
I'd like to use `tar_download` to download and store source data for my analysis. But when I try to run the pipeline, I get:

```
Error:
! Error running targets::tar_make()
  Target errors: targets::tar_meta(fields = error, complete_only = TRUE)
  Tips: https://books.ropensci.org/targets/debugging.html
  Last error: _paths_ no ETag or Last-Modified for url: [URL REDACTED]. Please choose a storage format other than format = "url" and pursue different workarounds to rerun the target when the remote data changes.
```

I understand that this function is designed to watch a remote source, but is there a way to disable that and just have it invalidate when the downloaded files are missing (other than writing a target manually)?

I was a bit confused because the error (and the [documentation for `tar_download`](https://docs.ropensci.org/tarchetypes/reference/tar_download.html)) implies that the `file` argument can be set, but it also sounds like that is fixed when using `tar_download` (since it returns two target objects - one with `format = "url"` and one with `format = "file"`).

Thanks!

**Comment by wlandau** [comment]
You could try `cue = tar_cue(file = FALSE)` in `tar_download()`, or you could use an ordinary `tar_target(..., command = tarchetypes::tar_download_run(), format = "file")`.

https://github.com/ropensci/tarchetypes/blob/4f75effd39342dd863fb6970fcbf54dcf92cbb13/R/tar_download.R#L158-L180
